src/titlepane/createTitlePaneElement.ts

```diff
@@ -0,0 +1,39 @@
+import { CustomElementDescriptor } from '@dojo/widget-core/customElements';
+import TitlePane from './TitlePane';
+
+/**
+ * Configures a TitlePane web component
+ */
+export default function createTitlePaneElement(): CustomElementDescriptor {
+	return {
+		tagName: 'dojo-title-pane',
+		widgetConstructor: TitlePane,
+		attributes: [
+			{
+				attributeName: 'headingLevel',
```

Comment:
Attribute names should be lowercase (there's a PR right now to update other component descriptors), so this should be:

```js
attributeName: 'headinglevel',
propertyName: 'headingLevel'
```